spec/index.bs

```diff
@@ -5914,6 +5915,12 @@ dictionary GPUImageCopyExternalImage {
     ::
         Defines the origin of the copy - the minimum corner of the source sub-region to copy from.
         Together with `copySize`, defines the full copy sub-region.
+    
+    : <dfn>originBottomLeft</dfn>
+    ::
+        Describes whether the source image is vertically flipped, or not.
+       
+        If this option is set to `true`, copy starts from the bottom row of the source.
```

Comment:
Should still clarify what `origin` means when this is set. I propose this for now, to be consistent with the naming of `originBottomLeft`:

```suggestion
        If this option is set to `true`, copy starts from the bottom row of the source region,
        and {{GPUImageCopyExternalTexture/origin}} is relative to the bottom-left
        corner, increasing upward.
```